CHANGELOG.md

```diff
@@ -18,7 +20,7 @@ populate, and send them to the Tessen service.
 - Added `sensuctl cluster id` which exposes the unique sensu cluster id.
 
 ### Changed
-- [Web] Updated embedded web assets from `275386a` ... `b0c1138`
+- [Web] Updated embedded web assets from `275386a` ... `46cd0ee`
```

Comment:
Is this true? The ref still says `b0c1138`

Reply:
Ah, cool  -- I may have done this wrong? I ran `go generate ./` in the `release/5.10.0` branch.

Reply:
yeah, my bad, I was looking at the master branch.